src/FSharp.Analyzers.SDK/FSharp.Analyzers.SDK.TestHelpers.fsi

```diff
@@ -0,0 +1,19 @@
+module FSharp.Analyzers.SDK.TestHelpers
+
+open FSharp.Compiler.CodeAnalysis
+
+type DotNetVersion =
+    | Six
+    | Seven
+
+    override ToString: unit -> string
+
+type FSharpProjectOptions with
+
+    static member zero: FSharpProjectOptions
+
+type Package = { Name: string; Version: string }
+
+val mkOptionsFromProject: version: DotNetVersion -> additionalPkg: Package option -> FSharpProjectOptions
```

Comment:
Maybe rename `version` to `framework and pass in a string instead?
Having a proper XML comment can point out what we are going to do with it.
`additionalPkg` should be a list.
The whole thing should be `Async<>` or `Task<>`